### PR TITLE
Pms v15 fix opf_bf tests and make_multiconductor!

### PR DESCRIPTION
**Pull request by frederikgeth**

### CODE REVIEW

File: src/form/bf.jl [hunk 1]
```diff
@@ -1,8 +1,9 @@
+
+
+
 ""
-function variable_mc_voltage(pm::LPLinUBFModel; kwargs...)
-    for cnd in _PMs.conductor_ids(pm)
-        _PMs.variable_voltage_magnitude_sqr(pm; cnd=cnd, kwargs...)
-    end
+function variable_mc_voltage(pm::LPLinUBFModel; n_cond::Int=3, nw::Int=pm.cnw, bounded = true)
```
Comment (sanderclaeys) [inline src/form/bf.jl]: what is the purpose of n_cond? It is not used and not being passed on either.

Comment (frederikgeth) [reply src/form/bf.jl]: ok, removed

File: src/prob/pf_bf.jl [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ""
 function run_mc_pf_bf(data::Dict{String,Any}, model_type, solver; kwargs...)
-    if model_type != SDPUBFPowerModel && model_type != SOCNLPUBFPowerModel && model_type != SOCConicUBFPowerModel && model_type != LPLinUBFPowerModel && model_type != LPdiagUBFPowerModel && model_type !=  SOCBFPowerModel
-        Memento.error(_LOGGER, "The problem type mc_opf_bf at the moment only supports a limited set of formulations")
-    end
+    # if model_type != SDPUBFPowerModel && model_type != SOCNLPUBFPowerModel && model_type != SOCConicUBFPowerModel && model_type != LPLinUBFPowerModel && model_type != LPdiagUBFPowerModel && model_type !=  SOCBFPowerModel
+    #     Memento.error(_LOGGER, "The problem type mc_opf_bf at the moment only supports a limited set of formulations")
+    # end
```
Comment (sanderclaeys) [inline src/prob/pf_bf.jl]: why commented?

Comment (frederikgeth) [reply src/prob/pf_bf.jl]: removed

**Pull request merged**